examples/asr/transcribe_speech.py

```diff
@@ -172,8 +172,9 @@ def main(cfg: TranscriptionConfig) -> TranscriptionConfig:
     if hasattr(asr_model, 'change_decoding_strategy'):
         # Check if ctc or rnnt model
         if hasattr(asr_model, 'joint'):  # RNNT model
-            rnnt_decoding = RNNTDecodingConfig(fused_batch_size=-1, compute_langs=cfg.compute_langs)
-            asr_model.change_decoding_strategy(rnnt_decoding)
+            cfg.rnnt_decoding.fused_batch_size = -1
```

Comment:
Rebase your pr, I've merged the other pr with these changes.

Reply:
This is done